package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ant-design/icons": "^4.6.3",
```

Comment:
you dont need to push to your git repo auto generic files

Reply:
What do you mean? This is dependency for folder form node modules and I don`t push node modules